spyder/plugins/profiler/widgets/main_widget.py

```diff
@@ -173,13 +174,21 @@ def __init__(self, name=None, plugin=None, parent=None):
         self.filecombo = PythonModulesComboBox(
             self, id_=ProfilerWidgetMainToolbarItems.FileCombo)
         self.datatree = ProfilerDataTree(self)
+        self.pane_empty = PaneEmptyWidget(
+            self,
+            "code-profiler",
+            _("You haven't profiled any code yet."),
+            _("Profile your code to find the run time and number of calls "
+              "for every function and method called on it.")
```

Comment:
```suggestion
            _("Profile your code to explore which functions and methods "
              "were called the most and took the longest to run, "
              "and find out where to optimize it.")
```

* Fix multiple grammar and semantic errors
* Mention the core reason _why_ users want to do this

Reply:
Small counterproposal to mention time to run first, which I think is what people associate most of the time with the idea of profilers.

```suggestion
            _("Profile your code to explore which functions and methods "
              "took the longest to run and were called the most, "
              "and find out where to optimize it.")
```

Reply:
Sure, that's better—I actually thought about it originally, but looks like I didn't actually make the change. I updated my above comment too to ensure it gets applied either way.